oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/RulesReader.java

```diff
@@ -264,16 +268,16 @@ private void readFeishuConfig(Rules rules) {
     @SuppressWarnings("unchecked")
     private void readWeLinkConfig(Rules rules) {
         Map<String, Object> welinkConfig = (Map<String, Object>) yamlData.getOrDefault(
-            "welinkHooks",
-            Collections.EMPTY_MAP
+                "welinkHooks",
```

Comment:
Unexpected change, please revert and polish changes.

Reply:
@wu-sheng yes, now its only a simply translate.

Reply:
i would be conside i18n implements deeply ,now i close this pr.